server/lyft/command/feature_runner.go

```diff
@@ -6,10 +6,76 @@ import (
 	"github.com/runatlantis/atlantis/server/core/config/valid"
 	"github.com/runatlantis/atlantis/server/events"
 	"github.com/runatlantis/atlantis/server/events/command"
+	"github.com/runatlantis/atlantis/server/events/models"
 	"github.com/runatlantis/atlantis/server/logging"
 	"github.com/runatlantis/atlantis/server/lyft/feature"
+	"github.com/runatlantis/atlantis/server/neptune/template"
 )
 
+type Commenter interface {
+	CreateComment(repo models.Repo, pullNum int, comment string, command string) error
+}
+
+type LegacyApplyCommentInput struct{}
+
+type PlatformModeRunner struct {
+	command.Runner
+	Allocator      feature.Allocator
+	Logger         logging.Logger
+	Builder        events.ProjectApplyCommandBuilder
+	TemplateLoader template.Loader[LegacyApplyCommentInput]
+	VCSClient      Commenter
+}
+
+func (a *PlatformModeRunner) Run(ctx *command.Context, cmd *command.Comment) {
+	if cmd.Name != command.Apply {
```

Comment:
nit: do we need this check if we're wrapping the applyCommandRunner? Or, should we just return instead of forwarding it downstream?

Reply:
this is a safety net in case we wrap the wrong one since it's all the same interface.